lib/dicom/binary_format.ex

```diff
@@ -574,4 +609,18 @@ defmodule Dicom.BinaryFormat do
 
     serialized
   end
+
+  def serialize_data_data_set(data_set) do
+    serialization_options = [endianness: :little, explicit: true]
```

Comment:
I think this method can simply be called `serialize`, as it will be the normal serialization method. The `serialize_command_dataset` is a special case.

Also please already move the `options` into the function signature and hard-code them in the association for now. Then the API here is already "final".